lib/enumerable/none.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_none?
+    raise 'Please provide a block' unless block_given?
+
+    result = true
```

Comment:
- [ ] my_none? when no block or argument is given returns true only if none of the collection members is true
- [ ] my_none? when a class is passed as an argument returns true if none of the collection is a member of such class
- [ ] my_none? when a Regex is passed as an argument returns true only if none of the collection matches the Regex
- [ ] my_none? when a pattern other than Regex or a Class is given returns true only if none of the collection matches the pattern

Reply:
```ruby
def my_none?(pattern = nil, &a_block)
    !my_any?(pattern, &a_block)
  end
```
This way `#my_any` solves all the problems

Solved!!